Doc/tutorial/errors.rst

```diff
@@ -301,8 +301,13 @@ This can be useful when you are transforming exceptions. For example::
 
 The expression following the :keyword:`from` must be either an exception or
 ``None``. Exception chaining happens automatically when an exception is raised
-inside an exception handler or :keyword:`finally` section. Exception chaining
-can be disabled by using ``from None`` idiom:
+inside an exception handler or :keyword:`finally` section, however in this case
+``__context__`` attribute of the exception is used. Chaining with
```

Comment:
I recommend submitting this sentence right before the builtin exceptions sentence after #23162 lands. Then add "Chaining with ... takes priority."